<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signinwindow</class>
 <widget class="QWidget" name="signinwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="regis_username">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="regis_pwd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="regis_pwd_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请再次确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="regis_email">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入邮箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Register">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="regis_phonenum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入手机号码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
